pkg/user/tx_client.go

```diff
@@ -329,6 +342,10 @@ func (client *TxClient) BroadcastTx(ctx context.Context, msgs []sdktypes.Msg, op
 	// might not always call ConfirmTx().
 	client.pruneTxTracker()
 
+	if !client.trustedMode {
+		return nil, fmt.Errorf("non-trusted mode not yet implemented")
+	}
```

Comment:
In theory, you should still be able to broadcast a transaction, it just wouldn't be resubmitted if the nonce was incorrect, it would instead just error back to the user.

Reply:
Why is this check necessary? It doesn't seem possible to set trustedMode to `false`.